.github/blog-template-demo.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: ''
+Author: Your name here
+Author handle: Any links you want associated with your name
+Date: ''
+Description: one line about your blog
+Folder with media files: ''
+Blog URL: ''
+Labels: Releases, Steering committee, Demo, Events
+Reviewers: ''
+
+---
+| Reviewer  | Date | Approval |
+| ------------- | ------------- | ------------- |
+| <!-- Your Github handle here -->  |   |:+1:, :monocle_face:, :-1:  |
+| <!-- Your Github handle here --> |   |  |
+
+
+## Blog draft
+<!-- Introduction about the feature you are going to explain, how you use it, and why others can find it useful -->
+
+### Step 1: [One sentence about this step]
```

Comment:
Not sure why we have steps here?

Reply:
In my experience, demos usually walk people through a series of steps to achieve a specific result. This template is supposed to work as a skeleton for the authors to fill out with the actual content. I can add a comment about steps being optional if you think this can generate some confusion.

Reply:
I think including a comment that this optional would be helpful.